README.md

```diff
@@ -584,15 +692,39 @@ Right?!
-
-This is bad for a number of reasons, chiefly that's it's darn difficult to read! Maybe we understand it well enough right now, but imagine being a complete outsider (or yourself, six months from now) and trying to discern the flow of information through that mess of a nested control flow. It's also unnecessarily difficult to debug, as several distinct logical processes are coupled together within a single, monolithic function. If there's an error in the code that checks the amount of money inserted, it might not show up until the function returns its final message. If we had that checking logic isolated in its own function, we could test it in isolation from the rest of the code. However, as currently constructed, we'd have to comb through the behemoth line-by-line.
-
-Last, but not least, there is **so much repeated code** within our current `vendingMachine()` function. It's not at all [DRY][dry]. Every single `case` has the exact same structure; the only differences are the name and cost of the snack. Let's start modularizing our code a bit better, extracting some pieces of logic into separate functions.
-
-### `validateSelection()`
-For our first reorganization, we're going to extract the logic that's handled by the `default` in our massive `switch` statement. Currently, if an invalid selection is made, none of the `case`s in the `switch` will match. We'll eventually end up in the `default`, which then returns the string `'Please select a valid snack.'`. Instead, let's create a new function with a single responsibility — checking if the selection is valid:
+#### Refactoring into many functions
+Our function, while now wonderfully reusable, is still massively
+overcomplicated. Remember, each function should have one responsibility, and
+ours currently has at least five: checking whether the snack selection is valid;
+getting the price of the snack selection; checking whether enough money was
+inserted; calculating the amount of change; constructing and returning the
+appropriate message after the transaction is complete.
+
+This is bad for a number of reasons, chiefly that's it's darn difficult to read!
+Maybe we understand it well enough right now, but imagine being a complete
+outsider (or your future self) and trying to discern the flow of information
+through that mess of a nested control flow. It's also unnecessarily difficult to
+debug, as several distinct logical processes are coupled together within a
+single, monolithic function. If there's an error in the code that checks the
+amount of money inserted, it might not show up until the function returns its
+final message. If we had that checking logic isolated in its own function, we
+could test it in isolation from the rest of the code. However, as currently
+constructed, we'd have to comb through the behemoth line-by-line.
```

Comment:
replace `through the behemoth` with `through this messy code` or something similar.